modules/gsqlite3backend/4.3.1_to_4.7.0_schema.sqlite3.sql

```diff
@@ -3,23 +3,9 @@
 PRAGMA foreign_keys = 0;
 
 BEGIN TRANSACTION;
-  CREATE TABLE domains_temp (
-    id                    INTEGER PRIMARY KEY,
-    name                  VARCHAR(255) NOT NULL COLLATE NOCASE,
-    master                VARCHAR(128) DEFAULT NULL,
-    last_check            INTEGER DEFAULT NULL,
-    type                  VARCHAR(8) NOT NULL,
-    notified_serial       INTEGER DEFAULT NULL,
-    options               VARCHAR(65535) DEFAULT NULL,
-    catalog               VARCHAR(255) DEFAULT NULL,
-    account               VARCHAR(40) DEFAULT NULL
-  );
+  ALTER TABLE domains ADD options VARCHAR(65535) DEFAULT NULL;
+  ALTER TABLE domains ADD catalog VARCHAR(255) DEFAULT NULL;
```

Comment:
sorry, one more comment though:
```diff
-  type                  VARCHAR(6) NOT NULL,
+  type                  VARCHAR(8) NOT NULL,
```

that probably needs an alter column here?

Reply:
i think that means you do need the full temporary table thing, or a direct schema update :-(

Reply:
Just discussed this with @mind04 (in private) - sqlite doesn't care. Both are literally an alias for `TEXT`. So either we don't do it, or we do take the pain and go for TEXT now. I'm currently leaning towards sticking with the current version, even if it would look a bit weird.